core/docs/spark-qualification-tool.md

```diff
@@ -3,25 +3,13 @@ layout: page
 title: Qualification Tool
 nav_order: 8
 ---
-# Qualification Tool
+# Spark Rapids Qualification Tool
```

Comment:
Do we actually need to name it `Spark Rapids Qualification`?
I will be more careful with changing section headers because this could break cross-references between files